runtime/lua/vim/filetype.lua

```diff
@@ -862,6 +897,12 @@ local extension = {
       return "text"
     end
   end,
+  cmd = function (path, bufnr)
+    if getline(bufnr, 1):match("^/\\*") then
```

Comment:
```suggestion
    if getline(bufnr, 1):find("^/\\*") then
```

Reply:
Although `^` means "start of line", not "first non-whitespace", so `getline(bufnr,1):sub(1,2) == "/%*"` should do the trick as well.

Reply:
Now it works correctly